src/main/java/uk/gov/justice/digital/delius/jpa/standard/repository/CourtReportDocumentRepository.java

```diff
@@ -10,6 +10,6 @@
 import java.util.List;
 
 public interface CourtReportDocumentRepository extends JpaRepository<CourtReportDocument, Long>, JpaSpecificationExecutor<CourtReportDocument> {
-    @Query("select document from CourtReportDocument document, CourtReport entity where document.courtReport = entity and document.offenderId = :offenderId")
+    @Query("select document from CourtReportDocument document, CourtReport entity where document.courtReport = entity and document.offenderId = :offenderId and document.softDeleted = 0")
```

Comment:
Just missed a non-obvious change. Court reports also get retrieved using a Filter:
uk.gov.justice.digital.delius.jpa.filters.CourtReportDocumentFilterTransformer  (this is to support filtering of court reports, which is the exception).
So I think you just need to add something like 


`
        predicateBuilder.add(criteriaBuilder.equal(documentRoot.get("offenderId"), offenderId));
        **predicateBuilder.add(criteriaBuilder.equal(documentRoot.get("softDeleted"), 0));**

`

Reply:
As far as I can see that filter is only used in the context of results returned from the repository in any case, in DocumentService. Not to say that we shouldn’t do it. It’s just that testing that change through DocumentServiceTest is odd because it means changing one of the repo mocks to return a soft deleted record which won't happen any more. 

Unfortunately there was no existing `CourtReportDocumentFilterTransformerTest` so will add one